dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -1046,43 +933,82 @@ public Response fixConflictsFromRemote ( @Context final HttpServletRequest reque
             Logger.error( this.getClass(), "Error fixing "+type+" conflicts from remote", e );
             return response( "Error fixing "+type+" conflicts from remote" , true );
         } finally {
-			try {
-				if (requesterEndPoint != null) {
-					// Discard conflicts if successful or failed
-					integrityUtil.discardConflicts(requesterEndPoint.getId(),
-							IntegrityType.valueOf(type.toUpperCase()));
-				}
-			} catch (DotDataException e) {
-				Logger.error(this.getClass(), "ERROR: Table "
-						+ IntegrityType.valueOf(type.toUpperCase()).getResultsTableName()
-						+ " could not be cleared on end-point [" + requesterEndPoint.getId()
-						+ "]. Please truncate the table data manually.", e);
-			}
-
-			HibernateUtil.closeSessionSilently();
-		}
+            try {
+                if (remoteIP != null) {
+                    // Discard conflicts if successful or failed
+                    integrityUtil.discardConflicts(remoteIP,
+                            IntegrityType.valueOf(type.toUpperCase()));
+                }
+            } catch (DotDataException e) {
+                Logger.error(this.getClass(), "ERROR: Table "
+                        + IntegrityType.valueOf(type.toUpperCase()).getResultsTableName()
+                        + " could not be cleared on request id [" + remoteIP
+                        + "]. Please truncate the table data manually.", e);
+            }
+
+            HibernateUtil.closeSessionSilently();
```

Comment:
I think this should be handle by annotations

Reply:
done https://github.com/dotCMS/core/pull/19273/commits/303cd0bf044e9265865287ea596719a4346e2895#diff-1c1ba904ed03287c1c0b48a6f3971449L949